netmaster/main.go

```diff
@@ -62,10 +63,14 @@ func parseOpts(opts *cliOpts) error {
 		"cluster-store",
 		"etcd://127.0.0.1:2379",
 		"Etcd or Consul cluster store url.")
+	flagSet.StringVar(&opts.controlURL,
+		"control-url",
```

Comment:
Specific reason for the suggestion? I think control-url suggests using this for exchanging control protocol information. If you have a strong UX sense for naming it as internal-url, we can discuss and make the change.